models/migrations/v144.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"code.gitea.io/gitea/models"
```

Comment:
This will soon fail the go vet, please don't import models directly. Instead for `models.MATRIX` you could use a local variable and put a comment where it comes from.